src/styles/main.scss

```diff
@@ -1,14 +1,43 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+@import "./blocks/stars.scss";
+@import "./blocks/cards.scss";
+@import "./blocks/header.scss";
+
 html {
-  font-family: Roboto, sans-serif;
+  font-family: "Roboto", sans-serif;
 }
 
 body {
   margin: 0;
 }
 
-* {
-  box-sizing: border-box;
+.catalog__cards {
+  display: grid;
+  grid-template-columns: 200px 200px 200px 200px;
+  column-gap: 48px;
+  row-gap: 47px;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+  padding-top: 50px;
+  padding-bottom: 71px;
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .catalog__cards {
+    grid-template-columns: 200px 200px 200px;
+  }
 }
 
-@import "_header.scss";
-@import "_catalog.scss";
+@media (min-width: 488px) and (max-width: 767px) {
+  .catalog__cards {
+    grid-template-columns: 200px 200px;
+  }
+}
+
+@media (max-width: 487px) {
+  .catalog__cards {
+    grid-template-columns: 200px;
+  }
+}
```

Comment:
looks like a BEM block. so move it to a separate file

Reply:
When I moved @media to a separate file  it does not work